### PR TITLE
更新

### PR DESCRIPTION
**Pull request by tianshidoubi**
更新了地名，以及工程列表和一些小杂项【顺带一提，perlin是什么意思我真搞不懂】

### CODE REVIEW

**Comment by lishaoxia1985** [comment]
what are you doing! why do you delete more translation items? are you serious?

**Comment by tianshidoubi** [comment]
> what are you doing! why do you delete more translation items? are you serious?
Oh, I'm so sorry, I just copied them and edited them in a notepad and pasted them. I didn't do anything outrageous, I'm sorry.

File: android/assets/jsons/translationsByLanguage/Simplified_Chinese.properties [hunk 1]
```diff
@@ -322,8 +313,7 @@ Declare war on [civName]? = 向[civName]宣战吗？
 [leaderName] of [nation] = [nation]的[leaderName]
 You'll pay for this! = 你会为此付出代价的！
 Negotiate Peace = 议和
- # Requires translation!
-Peace with [civName]? = 
+Peace with [civName]? = 与 [civName] 签署和平条约？
```
Comment (lishaoxia1985) [inline android/assets/jsons/translationsByLanguage/Simplified_Chinese.properties]: Chinese doesn't need the blank word, the blank is needed by other languages such as english, french and so on.

Comment (tianshidoubi) [reply android/assets/jsons/translationsByLanguage/Simplified_Chinese.properties]: > Chinese doesn't need the blank word, the blank is needed by other languages such as english, french and so on.

Oh,I know,I'm so sorry.

**Comment by tianshidoubi** [comment]
> what are you doing! why do you delete more translation items? are you serious?

And,I just delete "# Requires translation!",so...

**Comment by lishaoxia1985** [comment]
You delete more and more items which have been translated! you can look at the last of file.

File: android/assets/jsons/translationsByLanguage/Simplified_Chinese.properties [hunk 2]
```diff
@@ -296,22 +295,14 @@ Requires a [buildingName] in this city = 需要城市建有[buildingName]
...
-Queue empty = 
- # Requires translation!
-Add to queue = 
- # Requires translation!
-Remove from queue = 
- # Requires translation!
-Show stats drilldown = 
- # Requires translation!
-Show construction queue = 
+Current construction = 当前工程
```
Comment (lishaoxia1985) [inline android/assets/jsons/translationsByLanguage/Simplified_Chinese.properties]: in fact, the construction word includes buildings and armies, 当前项目 is a good translation. of cource so is the blow items about this.

**Comment by tianshidoubi** [comment]
> You delete more and more items which have been translated! you can look at the last of file.

Oh,fxxk,I'm sorry for my error,I'm going to edit them.
[And sorry,my English is not good]

File: android/assets/jsons/translationsByLanguage/Simplified_Chinese.properties (hunk 2 above)
Comment (tianshidoubi) [reply android/assets/jsons/translationsByLanguage/Simplified_Chinese.properties]: > in fact, the construction word includes buildings and armies, 当前项目 is a good translation. of cource so is the blow items about this.

【I think you are chinese,so I try to speak Chinese.】
哦，我并不这么想，实际上之前我也想翻译成“项目”，但总归觉得是有些蹩脚，于是我觉得...翻译成“工程”不错...也许你是对的吧。

**Pull request merged**